jellyfin-api/src/commonMain/kotlin/org/jellyfin/sdk/api/client/HttpClientOptions.kt

```diff
@@ -11,20 +14,20 @@ public data class HttpClientOptions(
 	val followRedirects: Boolean = true,
 
 	/**
-	 * Timeout for a complete HTTP request in milliseconds.
+	 * Timeout for a complete HTTP request in seconds.
```

Comment:
Because it's a duration type, there's no unit attached. I think you can simply remove that part (like so):

```suggestion
	 * Timeout for a complete HTTP request.
```